docs/index.rst

```diff
@@ -369,13 +378,3 @@ trackers, chat rooms, and mailing lists is expected to follow the
 .. _`no longer necessary if you are uploading to pypi.org`:
     https://packaging.python.org/guides/migrating-to-pypi-org/#registering-package-names-metadata
 .. _`no longer supported`: https://github.com/pypa/warehouse/issues/1627
-
-.. toctree::
-    :caption: Further documentation
-    :maxdepth: 3
-
-    contributing
-    changelog
-    Python Packaging User Guide <https://packaging.python.org/tutorials/distributing-packages/>
-
-* :ref:`search`
```

Comment:
Was this supposed to be removed?

Reply:
That was my intention; it seemed superfluous, given the more functional search box in the sidebar.

Reply:
Oh, fair enough. It's just not aligned with what the PR title advertised so I thought it was accidental.